CoTeTo/Loaders/XMLFile.py

```diff
@@ -0,0 +1,145 @@
+#-*- coding:utf-8 -*-
+#
+# This file is part of CoTeTo - a code generation tool
+# 20170608 Joerg Raedler jraedler@udk-berlin.de
+#
+# XML file loader for CoTeTo
+# 20180208 Werner Kaul we.kaul@udk-berlin.de'
+#
+from CoTeTo.Loader import Loader
+from os.path import isfile
+
+from lxml import etree, objectify
```

Comment:
This probably means lxml has to be added as a dependency in the setup.py I think. Or is it part of vanilla python, like the json and pickle reader?